about/aboutStyle.css

```diff
@@ -0,0 +1,62 @@
+body{
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  border-radius: 0;
+  border: none;
+  outline: none;
+  background: none;
+}
+
+.mainABOUT{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  width: 100%;
+  height: auto;
+  margin: auto;
+}
+.headerABOUT{
+  color: white;
+  background-color: black;
+  text-align: center;
+}
+.imageABOUT{
+  border: black 2px solid;
+  width: fit-content;
+  margin: 28 auto;
+  padding: 20;
+}
+.statment1{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 20 300;
```

Comment:
okay, I see why you did this. It looks great on desktop but you didn't add a media query to make sure these sections fill up the width of a mobile device. my suggestion is to actually do away with the margin 300. I'd make the width of the container 300px instead.